fluentui_topappbars/src/main/java/com/microsoft/fluentui/tokenized/AppBar.kt

```diff
@@ -140,7 +141,9 @@ fun AppBar(
                 if (appBarSize != AppBarSize.Large && navigationIcon.isIconAvailable()) {
                     Icon(
                         navigationIcon,
-                        modifier = Modifier
+                        modifier =
+                        Modifier
+                            .then(navigationIconModifier)
```

Comment:
I believe the ripple should be shown by default. Please check if that's not the case & need to adapt as per user scenario

Reply:
Hey, I have added default ripple behavior as you suggested, and also provided this modifier incase user wants to override ripple color, radius, etc.